test-workflows/pipelines/generators/clusterloader/eks/tests/pipeline.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: loadtest
+spec:
+  workspaces:
+    - name: secrets
+    - name: config
+    - name: source
+    - name: results
+  params:
+  - name: cluster-name
+    description: The name of the EKS cluster you want to spin.
+  - name: eks-version
+    default: "1.20"
+    description: The EKS version to install.
+  - name: region
+    description: The region where the cluster is in.
+    default: "us-west-2"
+  - name: zones
+    description: The zones where the cluster is in.
+    default: "us-west-2a,us-west-2b,us-west-2c"
+  - name: nodegroup-name
+    default: "linux-nodes"
+    description: The name of the nodegroup of the cluster.
+  - name: node-type
+    default: "m5.xlarge"
+    description: The type of the EC2 instaces for the nodegroup of the cluster.
+  - name: desired-nodes
+    default: "100"
+    description: The desired number of nodes in the cluster.
+  - name: min-nodes
+    default: "1"
+    description: The minimum number of nodes in the cluster.
+  - name: max-nodes
+    default: "100"
+    description: The maximum number of nodes in the cluster.
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/kubernetes/perf-tests.git
+  - name: pods-per-node
+    description: "pod density"
+    default: "10"
+  - name: nodes-per-namespace
+    description:  "nodes per namespace to get created for load test "
+    default: "10"
+  - name: cl2-load-test-throughput
+    description: " throughput used for mutate operations"
+    default: "10"
+  tasks:
+  - name: create-eks-cluster
+    taskRef:
+      name: eks-cluster-create
+    params:
+      - name: cluster-name
+        value: '$(params.cluster-name)'
+      - name: eks-version
+        value: '$(params.eks-version)'
+      - name: region
+        value: '$(params.region)'
+      - name: zones
+        value: '$(params.zones)'
+      - name: nodegroup-name
+        value: '$(params.nodegroup-name)'
+      - name: node-type
+        value: '$(params.node-type)'
+      - name: desired-nodes
+        value: '$(params.desired-nodes)'
+      - name: min-nodes
```

Comment:
Thoughts on setting min/max/desired to the same value to remove complexity here? Or hardcode min=1

Reply:
I would like to leave this flexibility so one off manual runs with different values could be created instead of hardcoding them.